content/webapp/components/FeaturedCard/FeaturedCard.tsx

```diff
@@ -149,13 +149,9 @@ const FeaturedCardExhibitionBody = ({
           className={`${font('intr', 4)} no-margin no-padding`}
         >
           <>
-            <time dateTime={exhibition.start.toUTCString()}>
```

Comment:
Was there a reason for this one to be UTC? But I'm thinking this is also to align based on the docs that Danny shared on Slack

Reply:
I think this was a mistake – I can't think of why the start would be UTC and the end would be ISO.